src/DynamicSelect.php

```diff
@@ -27,6 +31,10 @@ public function meta()
             'options' => $this->getOptions($this->dependentValues),
             'dependsOn' => $this->getDependsOn(),
             'dependValues' => $this->dependentValues,
+            'placeholder' => $this->placeholder,
```

Comment:
Why not?
```
'placeholder' => __('Pick a value'),
```

Reply:
My thought was that it gives more freedom to customize the label - for example if a user wanted the label value to be "Choose an option" as in the default Select field.

Reply:
But now we don't have any default translations, I think you can use `withMeta()` to overwrite the translations:

```
DynamicSelect::make('Country', 'country')
    ->options(['US' => 'United States', 'UK' => 'United Kingdom'])
    ->withMeta([
        'placeholder' => 'Choose an option'
    ])
```

Reply:
Makes sense. I've changed it to use localization.

`->withMeta([
        'placeholder' => 'Choose an option'
    ])`
Doesn't work.

Reply:
It works perfectly here...

Reply:
It now works here as well. Thanks again.

Reply:
Nice, good to hear!